src/commands/lambda/flare.ts

```diff
@@ -30,6 +30,18 @@ const LOGS_DIRECTORY = 'logs'
 const FUNCTION_CONFIG_FILE_NAME = 'function_config.json'
 const ZIP_FILE_NAME = 'lambda-flare-output.zip'
 const LOG_STREAM_COUNT = 3
+const FULL_OBFUSCATION = '****************'
+const MIDDLE_OBFUSCATION = '**********'
+const NON_OBFUSCATED_ENV_VARS = new Set([
+  constants.SITE_ENV_VAR,
+  constants.LOG_LEVEL_ENV_VAR,
+  constants.LAMBDA_HANDLER_ENV_VAR,
+  constants.SERVICE_ENV_VAR,
+  constants.VERSION_ENV_VAR,
+  constants.ENVIRONMENT_ENV_VAR,
+  constants.EXTRA_TAGS_ENV_VAR,
+  constants.DOTNET_TRACER_HOME_ENV_VAR,
+])
```

Comment:
Note that boolean/number environment variables are not included since we don't obfuscate booleans and numbers. An example is `DD_TRACE_ENABLED`

Reply:
Let's move this array into `constants.ts`

Reply:
This is also missing some environmet variables.

Reply:
I added `CI_SITE_ENV_VAR` and `AWS_DEFAULT_REGION_ENV_VAR`. Is there any other missing environment variables? Everything else seems to be a boolean/number or a secret.